### PR TITLE
Cleaned Up If Commands

### PR DESCRIPTION
**Pull request by ob-ab**
Cleaned up following files:
- BeginIfCommand.cs
- BeginMutliIfCommand.cs
- ElseCommand.cs
- EndIfCommand.cs

### CODE REVIEW

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs [hunk 1]
```diff
@@ -1,69 +1,61 @@
...
-
-
-
     [Serializable]
-    [Attributes.ClassAttributes.Group("If Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to evaluate a logical statement to determine if the statement is true.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to check if a statement is 'true' or 'false' and subsequently take an action based on either condition. Any 'BeginIf' command must have a following 'EndIf' command.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command evaluates supplied arguments and if evaluated to true runs sub elements")]
+    [Group("If Commands")]
+    [Description("This command evaluates a group of combined logical statements to determine if the combined result of the statements is 'true' or 'false' and subsequently performs action(s) based on result.")]
```
Comment (CameronHerwig) [inline taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs]: Change to "based on the result."

**Review by CameronHerwig — CHANGES_REQUESTED** [review]

File: taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs (hunk 1 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs]: done

**Review by CameronHerwig — DISMISSED** [review]

File: taskt/Core/Automation/Commands/If Commands/BeginIfCommand.cs [hunk 2]
```diff
@@ -1,127 +1,117 @@
...
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Folder Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Web Element Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("GUI Element Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error Occured")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error Did Not Occur")]
-        [Attributes.PropertyAttributes.InputSpecification("Select the necessary comparison type.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Value**, **Window Name Exists**, **Active Window Name Is**, **File Exists**, **Folder Exists**, **Web Element Exists**, **Error Occured**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Condition Type")]
+        [PropertyUISelectionOption("Value")]
```
Comment (openbots-ff) [inline taskt/Core/Automation/Commands/If Commands/BeginIfCommand.cs]:
```suggestion
        [PropertyUISelectionOption("Value Compare")]
```

File: taskt/Core/Automation/Commands/If Commands/BeginIfCommand.cs [hunk 3]
```diff
@@ -601,20 +677,18 @@ private void ifAction_SelectionChangeCommitted(object sender, EventArgs e)
...
+            if (_recorderControl.Visible)
             {
-                RecorderControl.Hide();
+                _recorderControl.Hide();
             }
 
-
             switch (ifAction.SelectedItem)
             {
                 case "Value":
```
Comment (openbots-ff) [inline taskt/Core/Automation/Commands/If Commands/BeginIfCommand.cs]:
```suggestion
                case "Value Compare":
```

**Review by openbots-ff — CHANGES_REQUESTED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Review by openbots-ff — APPROVED** [review]

**Pull request merged**